Add CLI troubleshooting instructions (#70)
twilio/twilio-video-app-android: README.md

The passcode will expire after one week. To generate a new passcode:

1. Run `twilio rtc:apps:video:delete`.

> 2. Run `twilio rtc:apps:video:deploy --authentication passcode`.

Reviewer: I don't think we need two steps here. 

"The passcode will expire after one week. To generate a new passcode, run `twilio rtc:apps:video:deploy --authentication passcode --override`."

#### Troubleshooting The Twilio CLI

If any errors occur after running the [Twilio CLI commands](#deploy-twilio-access-token-server), then run the following commands:

1. Run `twilio plugins:update` to update the rtc plugin to the latest version.

> 1. Run `twilio rtc:apps:video:delete` to delete any problematic authentication servers.

Reviewer 2: ```suggestion
1. Run `twilio rtc:apps:video:delete` to delete any existing authentication servers.
```

The passcode will expire after one week. To generate a new passcode:

1. Run `twilio rtc:apps:video:delete`.
2. Run `twilio rtc:apps:video:deploy --authentication passcode`.

#### Troubleshooting The Twilio CLI

> If any errors occur after running the `twilio rtc:apps:video:deploy --authentication passcode` command, then run the following commands:

Reviewer 3: Should we be less specific and just say hey if you run into (basically any) CLI problem you can try these commands? Mainly thinking we could avoid the duplicate maintenance of this command syntax. If syntax changes not sure we will remember to change here. Not a big deal though.

> If any errors occur after running the `twilio rtc:apps:video:deploy --authentication passcode` command, then run the following commands:

Reviewer 2: ```suggestion
If any errors occur after running a [Twilio CLI RTC Plugin](https://github.com/twilio-labs/plugin-rtc) command, then try the following steps.
```

```
twilio plugins:update
twilio rtc:apps:video:delete
twilio rtc:apps:video:deploy --authentication passcode

> ```

Reviewer 2: ```
# Update to the latest RTC Plugin
twilio plugins:update

# Delete any existing applications
twilio rtc:apps:video:delete

# Retry deploy
twilio rtc:apps:video:deploy --authentication passcode
```

> The commands in the above snippet updates the rtc plugin to the latest version, deletes any problematic authentication servers, and deploys a new authentication server.

Reviewer 3: There seem to be 2 different styles used. Here we list out all the commands and then describe what they all do. In the deploy section we specify the command and describe what it does on the same line as needed. Not a big deal but probably better to be consistent unless there is a reason not to. It seems like the "Run abc command to do blah" approach works in more situations. Again pretty minor but thinking probably best to follow other pattern.

> The commands in the above snippet updates the rtc plugin to the latest version, deletes any problematic authentication servers, and deploys a new authentication server.

Reviewer 2: ```suggestion
```
